mlir/lib/ExecutionEngine/ROCm/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ endif()
 # lld header files.
 include_directories(${CMAKE_SOURCE_DIR}/external/llvm-project/lld/include)
 
+# Set compile-time flags for ROCm path.
+add_definitions(-D__ROCM_PATH__="${ROCM_PATH}")
+
+add_definitions(-DROCM_4_4_OR_OLDER=${USE_ROCM_4_4_OR_OLDER})
```

Comment:
Could you follow the existing convention (use underscores before/after of a make variable), instead of making two very similar names.

```cpp
-D__ROCM_4_4_OR_OLDER__=${ROCM_4_4_OR_OLDER}
```